multisig-wallet/project/multisig-contract/tests/functions/cancel_transaction.rs

```diff
@@ -0,0 +1,41 @@
+use crate::utils::{
+    abi_calls::{cancel_transaction, constructor, nonce},
+    test_helpers::{default_users, setup_env, DEFAULT_THRESHOLD},
+    VALID_SIGNER_PK,
+};
+
+mod success {
+
+    use super::*;
+
+    #[tokio::test]
+    async fn cancels_transaction() {
```

Comment:
Can we check that the tests also emit the correct logs? [Example](https://github.com/FuelLabs/sway-applications/pull/322/files#diff-3bf4205ff279244340076b24dce7c26d8ba6c8767c54a9c47566046f3d83c09eR37-R42)